### PR TITLE
Support multiple Semantics listeners

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @abarth

**Comment by abarth** [comment]
LGTM

File: sky/services/semantics/semantics.mojom [hunk 1]
```diff
@@ -53,7 +53,7 @@ struct SemanticGeometry {
 };
 
 [ServiceName="semantics::SemanticsClient"]
```
Comment (chinmaygarde) [inline sky/services/semantics/semantics.mojom]: Missed the service name

Comment (abarth) [inline sky/services/semantics/semantics.mojom]: You should probably just remove the ServiceName.  I don't think you intend someone to provide this interface via a ServiceProvider.

Comment (Hixie) [inline sky/services/semantics/semantics.mojom]: yeah good point.

**Pull request merged**